### ISSUE TITLE
[ENH] download project by url

### ISSUE THREAD

**Issue by AlbertDeFusco** [Anaconda-Platform/anaconda-project#328]
Add the ability for anaconda-project download to use full urls to project archives

**Comment by jlstevens** [comment]
This would be very useful as the `download` command can make it easy to fetch remote projects across platforms (i.e. including windows) and also automatically runs `unarchive`.

**Comment by AlbertDeFusco** [comment]
let's not forget to make a better progress bar

**Comment by jbednar** [comment]
Yes, both for the project download (which is probably quick) and more importantly for data downloads (which can be minutes!).

**Comment by mcg1969** [comment]
Thanks for opening this issue, and apologies for the long delay in responding.

Anaconda Project has not been actively developed in some time and has largely been superseded by newer alternatives:

* [pixi](https://pixi.sh/) and [uv](https://docs.astral.sh/uv/) represent the first truly successful successors.
* [conda-workspaces](https://github.com/conda-incubator/conda-workspaces) is a promising pixi-compatible alternative for users who prefer to stay closer to the conda ecosystem.

Going forward, work on this repository is focused on critical bug fixes and on tooling to help users migrate existing Anaconda Project projects to pixi (and, by extension, conda-workspaces). See the [Pixi support](https://anaconda-project.readthedocs.io/en/latest/pixi-support.html) page in our documentation for details on the `anaconda-project export-pixi` command, which can convert an existing `anaconda-project.yml` into a `pixi.toml` for you.

The `anaconda-project` package itself will remain available, so existing projects relying on its stable feature set can continue to be supported. However, we do not anticipate further enhancement work, and for that reason we are closing out the bulk of our open issue backlog — including this one.

If this issue describes a critical regression in the stable feature set (as opposed to an enhancement or an edge case in a feature we no longer plan to expand) and you have a minimal reproducer against a current release, please feel free to reopen and we will take another look.

Thank you for your interest in Anaconda Project.
